src/App.tsx

```diff
@@ -1,70 +1,19 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect } from "react";
 import "./App.css";
 import Dashboard from "./Pages/Dashboard/Dashboard.component";
-import { makeStyles } from "@material-ui/core/styles";
-import { useTranslation } from "react-i18next";
-import Button from "@material-ui/core/Button";
 import { BrowserRouter as Router, Route, Redirect } from "react-router-dom";
+import NavBar from "Layout/Navbar";
+import { useTranslation } from "react-i18next";
 
-const useStyles = makeStyles({
-  root: {
-    width: "90%",
-    padding: "0.5rem 0",
-    paddingLeft: "35%",
-    margin: "auto",
-    backgroundColor: "grey",
-  },
-  button: {
-    margin: "10px",
-  },
-});
 function App() {
-  const classes = useStyles();
   const { i18n } = useTranslation();
-  const [isEnglish, setEnglish] = useState(true);
-  const [isHindi, setHindi] = useState(false);
-  const [isGujrati, setGujrati] = useState(false);
 
-  const handleClick = (lang: any) => {
-    setEnglish(false);
-    setHindi(false);
-    setGujrati(false);
-    lang === "en" && setEnglish(true);
-    lang === "hi" && setHindi(true);
-    lang === "guj" && setGujrati(true);
-    i18n.changeLanguage(lang);
-  };
   useEffect(() => {
     i18n.changeLanguage(process.env.DEFAULT_LANGUAGE);
   }, []);
```

Comment:
is this required here in this file ... are you using it here in app.tsx ?

Reply:
Before this i had put this in inner component (NavBar) but value is not getting their of "process.env.DEFAULT_LANGUAGE" , here In "App.tsx"  we are getting value, and it's working fine.